modules/core/src/convert.cpp

```diff
@@ -5287,6 +5292,43 @@ static bool ocl_LUT(InputArray _src, InputArray _lut, OutputArray _dst)
 
 #endif
 
+#ifdef HAVE_OPENVX
+static bool openvx_LUT(Mat src, Mat dst, Mat _lut)
+{
+    if (src.type() != CV_8UC1 || dst.type() != src.type() || _lut.type() != src.type() || !_lut.isContinuous())
+        return false;
+
+    try
+    {
+        ivx::Context ctx = ivx::Context::create();
+
+        ivx::Image
+            ia = ivx::Image::createFromHandle(ctx, VX_DF_IMAGE_U8,
```

Comment:
consider the shorter code:
```cpp
ia = ivx::Image::createFromHandle(
    ctx,
    ivx::Image::matTypeToFormat(src.type()),
    ivx::Image::createAddressing(src),
    src.data );
```

And I'm actually thinking about adding `ivx::Image::createFromHandle(context, mat);` to ivx.hpp, but this require saving that Mat reference inside the created Image...